src/frontend/components/Button/ButtonLayout/index.spec.tsx

```diff
@@ -0,0 +1,161 @@
+import React from 'react';
+import { render, screen } from '@testing-library/react';
+
+import { ButtonLayout } from '.';
+
+describe('<ButtonLayout />', () => {
+  it('renders the Button component with its title only', () => {
+    render(
+      <ButtonLayout
+        label="Button"
+        reversedColor={'reversed-color'}
+        tintColor={'int-color'}
+        textColor={'text-color'}
```

Comment:
t**e**xtColor is before t**i**ntColor in alphabetical order (same comment for all the ButtonLayout of this file)

Reply:
c72b659